zebra-chain/src/equihash_solution.rs

```diff
@@ -15,24 +18,87 @@ const EQUIHASH_SOLUTION_SIZE: usize = 1344;
 ///
 /// The size of an Equihash solution in bytes is always 1344 so the
 /// length of this type is fixed.
-#[derive(Clone)]
-pub struct EquihashSolution([u8; EQUIHASH_SOLUTION_SIZE]);
+pub struct EquihashSolution(pub [u8; EQUIHASH_SOLUTION_SIZE]);
```

Comment:
We might want to change the array to a `Box<[u8; 1344]>` so that we don't accidentally cause tons of stack traffic.  This makes it impossible to create `const` or `static` `EquihashSolution`s, but we can still use `lazy_static` to do it.

Reply:
(Note: also not related to this change, just an observation while looking at the source)

Reply:
#219